caffe2/core/predictor.cc

```diff
@@ -49,13 +49,16 @@ const NetDef& getNet(const MetaNetDef& def, const std::string& name) {
 
 const ::google::protobuf::RepeatedPtrField<::std::string>& getBlobs(
     const MetaNetDef& def,
-    const std::string& name) {
+    const std::string& name,
+    const bool optional = false) {
   for (const auto& b : def.blobs()) {
     if (b.key() == name) {
       return b.value();
     }
   }
-  CAFFE_THROW("Blob not found: ", name);
+  CAFFE_ENFORCE(optional, "Blob not found: ", name);
```

Comment:
Why do you need to make this `CAFFE_ENFORCE` optional?

Reply:
hm, agree with @hlu1 , this seems weird. Could you try getting rid of this, please?

Reply:
@hlu1 confirmed that real models should always have the outputs blob, so updated predictor_test.cc instead.